index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en-US">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Halogen</title>
```

Comment:
Hey dude! Good work! I think design-wise we want to keep the halogen name all lowercase. Do you think the title should be lowercase too?

![image](https://user-images.githubusercontent.com/13723156/106363716-e4c8c200-62ef-11eb-8767-d7eaec1d74c9.png)

Reply:
Good idea <3